src/main/java/io/jenkins/plugins/artifact_manager_s3/JCloudsArtifactManager.java

```diff
@@ -77,8 +76,7 @@
 /**
  * Artifact manager that stores files in a JClouds BlobStore using any of JClouds supported backends
  */
-@Restricted(Beta.class)
-public class JCloudsArtifactManager extends ArtifactManager implements StashManager.StashAwareArtifactManager {
+class JCloudsArtifactManager extends ArtifactManager implements StashManager.StashAwareArtifactManager {
```

Comment:
Works I think—the _factory_ must be `public` but the individual manager need not be.